scripts/ModifyPSProfile.psm1

```diff
@@ -35,9 +98,15 @@ function RemovefromProfile {
         [string] $Content
     )
 
-    ((Get-Content -Path $profile -raw) -replace "[\r\n]*$content",'').trim() | Set-Content $profile -NoNewLine
+    ((Get-Content -Path $PROFILE -raw) -replace "[\r\n]*$Content",'').trim() | Set-Content $PROFILE -NoNewLine
```

Comment:
```suggestion
    if ($null -eq (Get-Content -Path $PROFILE -raw)) { return }
    ((Get-Content -Path $PROFILE -raw) -replace "[\r\n]*$Content",'').Trim() | Set-Content $PROFILE -NoNewLine
```
When testing this script, I noticed that when the content in `$PROFILE` is empty or only contains spaces and line seperators, the return value of `(Get-Content...)` will be **$null** rather than an empty string.
This adds an extra check to avoid errors.

By the way, [this is why I put $null in the left](https://evotec.xyz/the-curious-case-of-null-should-be-on-the-left-side-of-equality-comparisons-psscriptanalyzer/).